h2o-core/src/main/java/water/parser/FVecParseReader.java

```diff
@@ -32,4 +32,5 @@ public FVecParseReader(Chunk chk){
   @Override public long getGlobalByteOffset(){
     return _goffset;
   }
+  public Chunk getChunk() { return _chk; }
```

Comment:
I added the javadoc. I don't think there is much harm in exposing the chunk. Users of FVecParseReader are already aware of the underlying implementation.